source/localizable/smart_cart/index.html.md.erb

```diff
@@ -0,0 +1,58 @@
+---
+parent: smart_cart
+title: overview
+---
+
+# Overview <%= edit_link %>
+
+There are currently two tools for Merchants that can be used to integrate Smart Cart orders
+into their existing system, the [Orders API](/smart_cart/orders_api) and
+[Webhooks](/smart_cart/webhooks).
+
+<%= partial 'partials/toc' %>
+
+## Orders API
+
+Smart Cart Orders API allows Skroutz Merchants **perform actions** on their Smart Cart orders,
+such as accepting or rejecting an order.
+
+[Orders API documentation](/smart_cart/orders_api)
+
+## Webhooks
+
+Smart Cart webhooks service allows Skroutz Merchants to **get notified** about new or updated
```

Comment:
Now that we decided that only one webhook is registered, I suggest altering webhooks with webhook and stop referring as service.